lighthouse-core/gather/driver.js

```diff
@@ -277,9 +277,17 @@ class Driver {
     return new Promise((resolve, reject) => {
       this.sendCommand('Page.getAppManifest')
         .then(response => {
-          // We're not reading `response.errors` however it may contain critical and noncritical
-          // errors from Blink's manifest parser:
-          //   https://chromedevtools.github.io/debugger-protocol-viewer/tot/Page/#type-AppManifestError
+          // handle errors
+          if (response.errors) {
+            response.errors.forEach(error => {
+              log.warn('driver', 'Manifest error', error.message);
```

Comment:
i prefer warning on all for now. can quiet it if its annoying. But right now the repeated debugStrings in the report are the more annoying thing.